tests/testthat/test-tab_footnote.R

```diff
@@ -780,7 +780,8 @@ test_that("Footnotes with no location are rendered correctly", {
 
   gt_footnotes_1 <-
     gt_tbl %>%
-    tab_footnote(footnote = "A footnote.")
+    tab_footnote(footnote = "A footnote.") %>%
+    tab_options(latex.use.longtable = TRUE)
```

Comment:
In test-tab_footnote and test-tab_stub_indent, could you remove the changes? We are testing html output here. If you want to add more tests for latex output, please do so in test-l_* .

Generally, please remove the changes where you are not testing for LaTeX output specifically, and feel free to create new tests for things you may want to test

Reply:
Regarding the option `latex.use.longtable`, could we rename this slightly to `latex.use_longtable`? This is just for consistency with the idea that dots separate either locations, contexts, or actions. It seems to me that `use_longtable` is a single concrete action here.